Build failure for develop branch on Ubuntu

Getting an error building GridPACK on Ubuntu-EIOC8. Cannot link hadrec application with the new block parser stuff.

``` 
[ 99%] Linking CXX executable hadrec.x
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ZoneParser33::~ZoneParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser34::TransformerParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::OwnerParser33::~OwnerParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::CaseParser33::CaseParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser35::GeneratorParser35(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ImpedCorrParser33::~ImpedCorrParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser34::~GeneratorParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::FixedShuntParser33::FixedShuntParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser34::GeneratorParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SystemParser34::SystemParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser35::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::CaseParser34::~CaseParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::OwnerParser33::OwnerParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, double, int)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ImpedCorrParser34::~ImpedCorrParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::OwnerParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::MultiSectParser33::MultiSectParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::FixedShuntParser33::~FixedShuntParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ImpedCorrParser33::parse(gridpack::stream::InputStream&, std::map<int, boost::shared_ptr<gridpack::component::DataCollection>, std::less<int>, std::allocator<std::pair<int const, boost::shared_ptr<gridpack::component::DataCollection> > > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser35::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, double, int)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::MultiSectParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BranchParser34::~BranchParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::MultiSectParser33::~MultiSectParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BusParser33::~BusParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser34::~TransformerParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TwoTermParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SwitchedShuntParser33::SwitchedShuntParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TwoTermParser33::TwoTermParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::VSCLineParser33::VSCLineParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser34::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::InterAreaParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser35::~TransformerParser35()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::CaseParser34::parse(gridpack::stream::InputStream&, boost::shared_ptr<gridpack::component::DataCollection>&, double&, int&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SwitchedShuntParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser33::~TransformerParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser33::GeneratorParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ImpedCorrParser34::parse(gridpack::stream::InputStream&, std::map<int, boost::shared_ptr<gridpack::component::DataCollection>, std::less<int>, std::allocator<std::pair<int const, boost::shared_ptr<gridpack::component::DataCollection> > > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::MultiTermParser33::~MultiTermParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::VSCLineParser33::~VSCLineParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BranchParser33::BranchParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser35::~GeneratorParser35()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser34::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, double, int)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::FACTSParser33::FACTSParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::CaseParser34::CaseParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::MultiTermParser33::MultiTermParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::LoadParser33::LoadParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SystemParser34::~SystemParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BranchParser33::~BranchParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::LoadParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ImpedCorrParser34::ImpedCorrParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::CaseParser33::parse(gridpack::stream::InputStream&, boost::shared_ptr<gridpack::component::DataCollection>&, double&, int&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ImpedCorrParser33::ImpedCorrParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BranchParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::AreaParser33::AreaParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SystemParser34::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::FixedShuntParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SwitchedShuntParser35::SwitchedShuntParser35(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::MultiTermParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser35::TransformerParser35(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SwitchedShuntParser33::~SwitchedShuntParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SysSwitchParser34::~SysSwitchParser34()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SwitchedShuntParser35::~SwitchedShuntParser35()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::AreaParser33::parse(gridpack::stream::InputStream&, boost::shared_ptr<gridpack::component::DataCollection>&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::InterAreaParser33::InterAreaParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::GeneratorParser33::~GeneratorParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BusParser33::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&, double, int, int*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SysSwitchParser34::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ZoneParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::CaseParser33::~CaseParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TwoTermParser33::~TwoTermParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::FACTSParser33::~FACTSParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SwitchedShuntParser35::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::FACTSParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BranchParser34::BranchParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BranchParser34::parse(gridpack::stream::InputStream&, std::vector<boost::shared_ptr<gridpack::component::DataCollection>, std::allocator<boost::shared_ptr<gridpack::component::DataCollection> > >&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::ZoneParser33::ZoneParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::InterAreaParser33::~InterAreaParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::AreaParser33::~AreaParser33()'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::BusParser33::BusParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::TransformerParser33::TransformerParser33(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::VSCLineParser33::parse(gridpack::stream::InputStream&)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::SysSwitchParser34::SysSwitchParser34(std::map<int, int, std::less<int>, std::allocator<std::pair<int const, int> > >*, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, int> > >*, std::map<std::pair<int, int>, int, std::less<std::pair<int, int> >, std::allocator<std::pair<std::pair<int, int> const, int> > >*)'
/usr/bin/ld: ../modules/dynamic_simulation_full_y/libgridpack_dynamic_simulation_full_y_module.so.3.3: undefined reference to `gridpack::parser::LoadParser33::~LoadParser33()'
collect2: error: ld returned 1 exit status
make[2]: *** [applications/hadrec/CMakeFiles/hadrec.x.dir/build.make:130: applications/hadrec/hadrec.x] Error 1
make[1]: *** [CMakeFiles/Makefile2:3008: applications/hadrec/CMakeFiles/hadrec.x.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

--- a/src/applications/modules/dynamic_simulation_full_y/CMakeLists.txt
+++ b/src/applications/modules/dynamic_simulation_full_y/CMakeLists.txt
@@ -21,6 +21,7 @@ set(target_libraries
     gridpack_powerflow_module
     gridpack_components
     gridpack_stream
+    gridpack_block_parsers
     gridpack_partition
     gridpack_parallel
     gridpack_math
@@ -113,8 +114,7 @@ add_library(gridpack_dynamic_simulation_full_y_module
 gridpack_set_library_version(gridpack_dynamic_simulation_full_y_module)
 
 target_link_libraries(gridpack_dynamic_simulation_full_y_module
-  gridpack_dsmatrix_components
-  gridpack_powerflow_module
+  ${target_libraries}
   )
 
 # -------------------------------------------------------------

